src/core/tools/common/simple_logger.cpp

```diff
@@ -53,49 +53,35 @@ DSN_DEFINE_validator(stderr_start_level, [](const char *level) -> bool {
     return strcmp(level, "LOG_LEVEL_INVALID") != 0;
 });
 
+std::function<const std::string()> log_prefixed_message_func = []() {
+    int tid = ::dsn::utils::get_current_tid();
+    char res[25];
+    sprintf(res, "unknow.io-thrd.%05d: ", tid);
+    return std::string(res);
+};
+
+void set_log_prefixed_message_func(std::function<const std::string()> func)
```

Comment:
```suggestion
void set_log_prefixed_message_func(std::function<std::string()> func)
```

Place this function in `logging.cpp`. Because log_prefixed_message_func is not limited to simple_logger, it's shared with other implementations (though we do not have now).